go/pkg/error_code/unkey_database_not_unique.go

```diff
@@ -10,8 +10,8 @@ func NewUnkeyDatabaseNotUniqueError(err error) UnkeyDatabaseNotUniqueError {
 			err,
 			SystemUnkey,
 			NamespaceKey,
-			"NOT_UNIQUE",
```

Comment:
Not sure if this should stay as NOT_UNIQUE, I thought it would be better if the error codes are the same everywhere

Reply:
I agree. I think it's safe to keep them same.

Reply:
🫡